src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -26,163 +26,1921 @@
+                    'href' => '#',
+                    'desc' => $this->trans('Save', 'Admin.Actions'),
+                ];
+                $back = Tools::safeOutput(Tools::getValue('back', ''));
+                if (empty($back)) {
+                    $back = $this->generateUrl('admin_features_index');
+                }
+                if (!Validate::isCleanHtml($back)) {
+                    die(Tools::displayError());
+                }
+                if (!$this->liteDisplay) {
+                    $this->toolbarButton['cancel'] = [
+                        'href' => $back,
+                        'desc' => $this->trans('Cancel', 'Admin.Actions'),
+                    ];
+                }
+
+                break;
+            case 'view':
+                // Default cancel button - like old back link
+                $back = Tools::safeOutput(Tools::getValue('back', ''));
+                if (empty($back)) {
+                    $back = $this->generateUrl('admin_features_index');
+                }
+                if (!Validate::isCleanHtml($back)) {
+                    die(Tools::displayError());
+                }
+                if (!$this->liteDisplay) {
+                    $this->toolbarButton['back'] = [
+                        'href' => $back,
+                        'desc' => $this->trans('Back to list', 'Admin.Actions'),
+                    ];
+                }
+
+                break;
+            case 'options':
+                $this->toolbarButton['save'] = [
+                    'href' => '#',
+                    'desc' => $this->trans('Save', 'Admin.Actions'),
+                ];
+
+                break;
+            default:
+                // list
+                //if ($this->allow_export) {
+                //    $this->toolbar_btn['export'] = [
+                //        'href' => self::$currentIndex . '&export' . $this->table . '&token=' . $this->token,
+                //        'desc' => $this->trans('Export', 'Admin.Actions'),
+                //    ];
+                //}
+        }
+    }
+
+    public function initPageHeaderToolbar()
+    {
+        if (empty($this->toolbarTitle)) {
+            $this->initToolbarTitle();
+        }
+
+        //Todo pas encore fait l'edition
+        //switch ($this->display) {
+        //    case 'view':
+        //        // Default cancel button - like old back link
+        //        $back = Tools::safeOutput(Tools::getValue('back', ''));
+        //        if (empty($back)) {
+        //            $back = self::$currentIndex . '&token=' . $this->token;
+        //        }
+        //        if (!Validate::isCleanHtml($back)) {
+        //            die(Tools::displayError());
+        //        }
+        //        if (!$this->lite_display) {
+        //            $this->page_header_toolbar_btn['back'] = [
+        //                'href' => $back,
+        //                'desc' => $this->trans('Back to list'),
```

Comment:
Don't you miss the translation domain here and for the following strings? It's Admin.Actions but maybe you specified it elsewhere in the code.